episodes/lesson-content.md

```diff
@@ -92,23 +93,6 @@ Finally, end your fenced div with a closing tag consisting of at least 3 colons
 and we will explore most of them in this episode.
 :::
```

Comment:
Think this needs to move into the PR section below.  I'm a bit concerned about introducing PRs and fenced divs at the same time.  Maybe instead, we have demo us creating a PR (and one of their team mebers) to add the prereqs as a bulleted list to that page with us. Then after reviewing it mention it would look better formatted and we can do that with fenced divs and then have another team member make the PR as an exercise.